cirq/testing/logs.py

```diff
@@ -20,7 +20,8 @@
 def assert_logs(
     *matches: str,
     count: Optional[int] = 1,
-    level: int = logging.WARNING,
+    min_level: int = logging.WARNING,
+    max_level: int = logging.CRITICAL,
```

Comment:
This should raise an error if `max_level < min_level`, as otherwise we'll silently pass assertions like this:
```
with assert_logs(max_level=logging.INFO, count=0):
    # will pass regardless of logs generated here
```